features/steps/thens/alignment.py

```diff
@@ -51,6 +51,23 @@ def is_3d(entity: ifcopenshell.entity_instance) -> bool:
         )
 
 
+def count_segments(logic, representation):
+    """
+    Used in ALA002 to return count of segments for business logic
+    and geometry representation.
+    """
+    try:
+        logic_count = len(logic.segments)
+    except AttributeError:
+        logic_count = None
+    try:
+        rep_count = len(representation.segments)
+    except AttributeError:
+        rep_count = None
+
+    return logic_count, rep_count
+
+
 @gherkin_ifc.step('A representation by {ifc_rep_criteria} requires the {ifc_layout_criteria} in the business logic')
```

Comment:
Can we split the `ifc_layout_criteria` up?  
So that we get 
`' A representation by {ifc_rep_criteria} requires the {existence:absence_or_presence} of {entities} in the business logic'`

To allow for only presence/absence as string input (on top in the file, similar as in values.py)
```
from parse_type import TypeBuilder
register_type(value_or_type=TypeBuilder.make_enum(dict(map(lambda x: (x, x), ("presence", "absence"))))) 
```